pkg/exchange/retry/order.go

```diff
@@ -47,6 +47,15 @@ func GeneralBackoff(ctx context.Context, op backoff2.Operation) (err error) {
 	return err
 }
 
+func GeneralBackoffLite(ctx context.Context, op backoff2.Operation) (err error) {
+	err = backoff2.Retry(op, backoff2.WithContext(
```

Comment:
Is it possible to rename `backoff2` to just `backoff`?

Reply:
simplify it.